packages/core/src/classes.ts

```diff
@@ -680,6 +680,10 @@ export interface BackupStatus {
 export interface BaseWorkspaceInfo {
   workspace: string // An uniq workspace name, Database names
   disabled?: boolean
+
+  archived?: boolean
```

Comment:
As an idea: extend `WorkspaceMode` with `archived` and use `lastProcessingTime` as an archived date when the mode is archived. Otherwise, we need to explicitly alter workspace table in postgres via a new migration to support the new fields. 
What do you think?